controller/rest/delete.go

```diff
@@ -84,18 +85,29 @@ func (s *Server) DeleteVolume(rw http.ResponseWriter, req *http.Request) error {
 		replicas DeletedReplicas
 		wg       sync.WaitGroup
 	)
+	logrus.Info("Delete volume")
 	apiContext := api.GetApiContext(req)
+	id := mux.Vars(req)["id"]
+	v := s.getVolume(apiContext, id)
+	if v == nil {
+		rw.WriteHeader(http.StatusNotFound)
```

Comment:
So here we set a 404 (not found) but in the above code we will say its a bad request. Its *not* a bad request -- the volume was simply not found

Reply:
Correct, will fix the log.